components/list/list-item-checkbox-mixin.js

```diff
@@ -25,7 +26,11 @@ export const ListItemCheckboxMixin = superclass => class extends SkeletonMixin(L
 			/**
 			 * Whether the item is selected
 			 */
-			selected: { type: Boolean, reflect: true }
+			selected: { type: Boolean, reflect: true },
+			/**
+			 * Private. The selection info (set by the selection component).
+			 */
+			selectionInfo: { type: Object }
```

Comment:
Should this have `attribute: false` so that people can't set the attribute in the HTML and have it bind?